lib/libevent/libevent.nit

```diff
@@ -458,6 +469,35 @@ extern class ConnectionListener `{ struct evconnlistener * `}
 	end
 end
 
+# A listener acting on an Unix Socket, spawns `Connection` on new connections
+extern class UnixConnectionListener `{ struct evconnlistener * `}
+	super ConnectionListener
+
+	private new bind_to(base: NativeEventBase, address: CString, file: CString, factory: ConnectionFactory)
+	import ConnectionFactory.accept_connection, error_callback `{
+		struct sockaddr_un sun;
+		struct evconnlistener *listener;
+
+		unlink(file);
+
+		ConnectionFactory_incr_ref(factory);
+		struct hostent *hostent = gethostbyname(address);
+
+		memset(&sun, 0, sizeof(struct sockaddr_un));
```

Comment:
If `sun` if safe as local structure, this can be replaced by a definition-time assignation.

```C
struct sockaddr_un sun = { 0 };
```

Reply:
Yup... I'll do it for `ConnectionListener::bind_to` as well